docs/rules/prefer-query-selector.md

```diff
@@ -1,6 +1,6 @@
 # Prefer `.querySelector()` over `.getElementById()`, `.querySelectorAll()` over `.getElementsByClassName()` and `.getElementsByTagName()`
 
-It's better to be consistent.
+It's better to be use one method to query DOM elements.
```

Comment:
I'm not a native speaker, but this sentence is not right to me

Reply:
My bad. Fixed it now.